recipes/somajo/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "somajo" %}
+{% set version = "2.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/SoMaJo-{{ version }}.tar.gz
+  sha256: 2d9cdf4fe191d75123a908038e84413407273150ce20242a2b1d1d2c1d1a49e9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - regex >=2019.02.18
+
+test:
+  imports:
+    - somajo
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/tsproisl/SoMaJo
+  summary: A tokenizer and sentence splitter for German and English web and social media texts.
+  license: GPL-3.0
```

Sorry, missed this earlier:
```suggestion
  license: GPL-3.0-or-later
```